Dockerfile

```diff
@@ -29,6 +29,10 @@ RUN apt-get update -y \
 
 WORKDIR /home/app/src
 
+ARG FONT_AWESOME_AUTH_TOKEN=""
```

Comment:
Better than a build arg would be a secret mount, so that the value doesn't get baked in to the image: https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/reference.md#run---mounttypesecret